scripts/autotask.js

```diff
@@ -0,0 +1,1901 @@
+const {
+  DefenderRelaySigner,
+  DefenderRelayProvider,
+} = require('defender-relay-client/lib/ethers');
+const ethers = require('ethers');
+const { hexZeroPad } = require('@ethersproject/bytes');
+const { parseUnits, formatUnits } = require('@ethersproject/units');
+const { KeyValueStoreClient } = require('defender-kvstore-client');
+
+const MARKET_ABI = [
```

Comment:
fwiw you only need the part of the abi which you use, so this could be reduced to just the following (assuming you can't just import the abi instead):

```ts
const MARKET_ABI = [
{
    inputs: [
      {
        internalType: 'address',
        name: 'recipient',
        type: 'address',
      },
      {
        internalType: 'uint256',
        name: 'amount',
        type: 'uint256',
      },
      {
        internalType: 'uint256',
        name: 'deadline',
        type: 'uint256',
      },
      {
        internalType: 'uint8',
        name: 'v',
        type: 'uint8',
      },
      {
        internalType: 'bytes32',
        name: 'r',
        type: 'bytes32',
      },
      {
        internalType: 'bytes32',
        name: 's',
        type: 'bytes32',
      },
    ],
    name: 'swap',
    outputs: [],
    stateMutability: 'nonpayable',
    type: 'function',
  }
]
```

Reply:
at some point our hope is to manage this autotask code with the defender client and thus incorporate it as part of our repo.... not sure if we'll ever be able to import the ABIs though.  the code in this repo is actually just for editing it in a decent text editor, I probably shouldn't actually commit this at all since we can always just go grab whatever the actual current state of the autotask code is from defender